<launch>
  <!-- zmp optimization paramters -->
  <group ns="xpp">  
    <param name="swing_time"          value="0.7"  type="double"/>
    <param name="stance_time"         value="0.2"  type="double" />	
    <param name="robot_height"        value="0.58" type="double"/> 
    <param name="stance_time_initial" value="0.4"  type="double"/> 
    <param name="stance_time_final"   value="0.2"  type="double"/> 
  </group>
</launch>
